safe_core/src/client/mock/tests.rs

```diff
@@ -663,18 +709,22 @@ fn mutable_data_permissions() {
     let data = SeqMutableData::new_with_data(name, tag, entries, Default::default(), owner_key);
     let address: MDataAddress = *data.address();
 
+    let token_signed_by_client = Some(test_generate_signed_token(client_safe_key.clone()));
+
     // Put it to the network.
     send_req_expect_ok!(
         &mut connection_manager,
         &client_safe_key,
+        token_signed_by_client.clone(),
```

Comment:
This operation is done by a user/client(and not by an app). A token being passed as a client will not get checked right? Why are we passing one here?

Reply:
> A token being passed as a client will not get checked right?
Right, we have that check for `is_app`

Reply:
Yup, so we don't have to pass a token here, just a `None` would do :)